components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -221,6 +221,34 @@ - (void)centerLayoutAnimated:(BOOL)animated {
   }
 }
 
+- (void)updateLabelVisibility {
+  if (self.selected) {
+    switch (self.titleVisibility) {
+      case MDCBottomNavigationBarTitleVisibilitySelected:
+        self.label.hidden = NO;
+        break;
+      case MDCBottomNavigationBarTitleVisibilityAlways:
+        self.label.hidden = NO;
```

Comment:
instead of `break` twice, would you be able to just put both cases one after the other?